examples/CantileverDistributedLoad/cantilever_conservative_distributed_load.py

```diff
@@ -0,0 +1,224 @@
+                self.callback_params["time"].append(time)
+                self.callback_params["step"].append(current_step)
+                self.callback_params["position"].append(
+                    system.position_collection.copy()
+                )
+                self.callback_params["com"].append(
+                    system.compute_position_center_of_mass()
+                )
+                self.callback_params["radius"].append(system.radius.copy())
+                self.callback_params["velocity"].append(
+                    system.velocity_collection.copy()
+                )
+                self.callback_params["avg_velocity"].append(
+                    system.compute_velocity_center_of_mass()
+                )
+
+                self.callback_params["center_of_mass"].append(
+                    system.compute_position_center_of_mass()
+                )
+                self.callback_params["velocity_magnitude"].append(
+                    (
+                        stretchable_rod.velocity_collection[-1][0] ** 2
+                        + stretchable_rod.velocity_collection[-1][1] ** 2
+                        + stretchable_rod.velocity_collection[-1][2] ** 2
+                    )
+                    ** 0.5
+                )
+
+    recorded_history = ea.defaultdict(list)
+    stretch_sim.collect_diagnostics(stretchable_rod).using(
+        CantileverDistributedLoadCallBack,
+        step_skip=200,
+        callback_params=recorded_history,
+    )
+
+    stretch_sim.finalize()
+    timestepper = ea.PositionVerlet()
+    # timestepper = PEFRL()
+
+    total_steps = int(final_time / dt)
+    ea.integrate(timestepper, stretch_sim, final_time, total_steps)
+
+    relative_tip_position = np.zeros(
+        2,
+    )
+    relative_tip_position[0] = (
+        find_tip_position(stretchable_rod, n_elem)[0] / base_length
+    )
+    relative_tip_position[1] = (
+        -find_tip_position(stretchable_rod, n_elem)[1] / base_length
+    )
+
+    if Animation:
+        plot_video_with_surface(
+            [recorded_history],
+            video_name="cantilever_conservative_distributed_load.mp4",
+            fps=rendering_fps,
+            step=1,
+            # The following parameters are optional
+            x_limits=(-0.0, 0.5),  # Set bounds on x-axis
+            y_limits=(-0.5, 0.0),  # Set bounds on y-axis
+            z_limits=(-0.0, 0.5),  # Set bounds on z-axis
+            dpi=100,  # Set the quality of the image
+            vis3D=True,  # Turn on 3D visualization
+            vis2D=False,  # Turn on projected (2D) visualization
+        )
+
+    relative_tip_position = np.zeros(
+        2,
+    )
+    relative_tip_position[0] = (
+        find_tip_position(stretchable_rod, n_elem)[0] / base_length
+    )
+    relative_tip_position[1] = (
+        -find_tip_position(stretchable_rod, n_elem)[1] / base_length
+    )
+
+    print(relative_tip_position)
+    return relative_tip_position
+
+
```

Comment:
Wrap everything below in
```Python
if __name__ == "__main__":
    "Run code here"
```
See [this case](https://github.com/GazzolaLab/PyElastica/blob/master/examples/ContinuumFlagellaCase/continuum_flagella.py#L156) for an example.